src/Extensions/ContentReviewDefaultSettings.php

```diff
@@ -222,9 +222,14 @@ public function getReviewFrom()
         if ($from) {
             return $from;
         }
-
         // Fall back to admin email
-        return Config::inst()->get(Email::class, 'admin_email');
+        $adminEmail = Config::inst()->get(Email::class, 'admin_email');
+        if (is_array($adminEmail)) {
+            // May be configured using an array ['admin-email@mysite.text' => 'Admin email label']
+            // https://docs.silverstripe.org/en/developer_guides/email/#administrator-emails
+            return array_values(array_keys($adminEmail))[0];
+        }
+        return $adminEmail;
```

Comment:
Maybe `Email::getDefaultFrom()` should be made public instead, and use that? Or else put that functionality in some other shared location?

Reply:
`Email::getDefaultFrom()` can return an array so that's no good here.

This logic of flattening an array isn't shared anywhere else.  (Symfony) Email can handle a email address as an array

Reply:
Ahh, looks like this has to be a string because of the CMS 5 implementation of `isValidEmail` which requires `?string`. That's unfortunate.... but it is what it is.